ffmpeg-plugin/build-ffmpeg.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2024 Intel Corporation
+
+set -e
+
+cd FFmpeg
+cp -f ../mcm_* ./libavdevice/
+
+make -j "$(nproc)"
+sudo make install
+cd ..
```

Comment:
```suggestion
cd "${SCRIPT_DIR}/FFmpeg"
cp -f "${SCRIPT_DIR}/mcm_"* "${SCRIPT_DIR}/FFmpeg/libavdevice/"

make -j "$(nproc)"
sudo make install
cd "${OLDPWD}"
```

Reply:
Let's improve the script in a separate PR. I suggest not to amend this one.